### PR TITLE
Hacks to change ant.design theme

### PR DESCRIPTION
**Pull request by miguelbrandao**
Well.... Pretty much the only way I think it can be done without messing with [deprecated plugins](https://www.npmjs.com/package/@zeit/next-less) and [disabling Next's built-in CSS support](https://nextjs.org/docs/messages/built-in-css-disabled)...

This solution is not very maintainable, but with a very obvious warning in the README I guess it could work for a while: it also has less chances of breaking current stuff and future stuff than [messing around with Webpack in `next.config.js`](https://nextjs.org/docs/api-reference/next.config.js/custom-webpack-config)

Let me know if it's worth it to pull this. If it is I'll add the warning to the README before merging

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,10 +3,12 @@
   "version": "0.1.0",
   "private": true,
   "scripts": {
-    "develop": "npm-run-all dev",
-    "dev": "next dev",
```
Comment (nelsonmestevao) [inline package.json:7]: I really like the dev shortcut, please don't take it away from me :pray:

File: styles/antd-theme.less [hunk 2]
```diff
@@ -0,0 +1,3 @@
+@import "../node_modules/antd/dist/antd.less";
+
+@primary-color: #642580;
```
Comment (nelsonmestevao) [inline styles/antd-theme.less:3]: We were using #722ED1 in the figma design that's a bit more light, and I think looks better.

```suggestion
@primary-color: #722ED1;
```

File: styles/globals.css [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import "~antd/dist/antd.css";
+@import "~styles/antd-theme.css";
```
Comment (nelsonmestevao) [inline styles/globals.css:1]:
```suggestion
@import "~/styles/antd-theme.css";
```

**Comment by nelsonmestevao** [comment]
It requires so little change that I think it's worth it. We can revert if we change our minds in the future. Maybe it will be even cleaner if Next.js gets LESS support again like [this PR](https://github.com/vercel/next.js/pull/23185) is proposing.

File: package.json (hunk 1 above)
Comment (miguelbrandao) [reply package.json:7]: Ahahah, I'll fix it in the next commit :+1:

**Comment by miguelbrandao** [comment]
I fixed the things you pointed out @nelsonmestevao.

I also added the warning to the README and customized the border radius of Ant, in order to be a bit more in harmony with the Tailwind styling.

I think its ready for merging.

**Review by nelsonmestevao — APPROVED** [review]
Well done! 💪 Really liked the readme warning 😅

**Pull request merged**